src/hooks/useMapClickHandlers.ts

```diff
@@ -18,7 +18,8 @@ export function useMapClickHandlers() {
         | MarkerData
         | AhbapData
         | TeleteyitData
-        | SahraKitchenData,
+        | SahraKitchenData
+        | any,
```

Comment:
Why did you need to add `any` here? Because `any` will override unions and `selectedMarkerData` will be `any`.